config/application.rb

```diff
@@ -39,3 +39,18 @@ class Application < Rails::Application
     config.cas_url = "https://login.umd.edu/cas"
   end
 end
+
+module Blacklight::Solr
+  class Repository < Blacklight::AbstractRepository
+    protected
+      # set the SSL CA certs path for Solr connections
+      def build_connection
+        if Rails.configuration.ssl_ca_file
```

Comment:
@peichman-umd I think this line should be changed to `if Rails.configuration.respond_to?(:ssl_ca_file)` to avoid `undefined method `ssl_ca_file'`